### PR TITLE
media-libs/lv2: add python 3.10

### PR DESCRIPTION
**Pull request by t0b3**
Closes: https://bugs.gentoo.org/829440

### CODE REVIEW

**Comment by fordfrog** [comment]
this one does not pass the validation because of `dev-python/rdflib` dropping `ia64` keyword. not sure how to handle it, maybe we'd need to drop it too...

**Comment by t0b3** [comment]
I'm open for dropping ia64 if it's no longer available on the dependency
rdflib.

Miroslav Šulc ***@***.***> wrote on Sat., 18. Dez. 2021,
12:02:

**Comment by t0b3** [comment]
Hmm, dropping ~ia64 would impact all the depending ebuilds:
- lilv
- sratom
- ffmpeg
Maybe the best solution would be to add the ~ia64 KEYWORD to latest rdflib ebuils.

**Comment by t0b3** [comment]
depends on PR https://github.com/gentoo/gentoo/pull/23494

**Comment by arthurzam** [comment]
Well, adding ia64 to rdflib won't happen in near future (as it's dependency berkeleydb has ilegal instuction on it).
I propose a better solution which would be easiest and best in my opinion: mask doc use flag on ia64.

In `gentoo:/profiles/arch/ia64/package.use.mask` add a mask with comment and author for:
`media-libs/lv2 doc`
You can see an example made by me there for how it can look.

**Comment by arthurzam** [comment]
Please split this into two commits:

1. mask the doc USE
2. enable py3.10

File: profiles/arch/ia64/package.use.mask [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Thomas Bettler <thomas.bettler@gmail.com> (2021-12-24)
+# dev-python/rdflib isn't keyworded here. Waiting for bug #814179
+media-libs/lv2 doc
```
Comment (thesamesam) [inline profiles/arch/ia64/package.use.mask:6]: Yep!

Comment (thesamesam) [reply profiles/arch/ia64/package.use.mask:6]: Hm, it's unconditionally RDEPENDing on it too... https://gitweb.gentoo.org/repo/gentoo.git/tree/media-libs/lv2/lv2-1.18.2.ebuild#n38

Is that correct?

Comment (t0b3) [reply profiles/arch/ia64/package.use.mask]: Hmm, thank you for spotting this :) We need to take another solution...

**Comment by thesamesam** [comment]
On the one hand, this is quite a heavy-handed approach (it'd be better to just add 3.10 to the latest version in a new revision, then block cleanup until we have a resolution). On the other hand, not convinced anyone is actually using this on ia64 anyway.

**Comment by t0b3** [comment]
Result is now:
lv2-1.18.2 | PythonCompatUpdate | PYTHON_COMPAT update available: python3_10
lv2-1.18.2-r1 | DroppedKeywords | ia64

If you are fine with this, let's proceed ...

**Comment by t0b3** [comment]
@thesamesam let's go then w/o ia64?

**Pull request closed without merging**